private/buf/bufsync/commits_to_sync_test.go

```diff
@@ -130,6 +125,7 @@ func TestCommitsToSyncWithNoPreviousSyncPoints(t *testing.T) {
 type mockErrorHandler struct{}
 
 func (*mockErrorHandler) HandleReadModuleError(readErr *ReadModuleError) LookbackDecisionCode {
+	// fmt.Printf("read error: %v\n", readErr)
```

Comment:
Is this comment intentionally left here?

Reply:
Yes, same as L107, useful sometimes when debugging the test. I think I'll just remove them.

Reply:
Done